Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -3519,48 +3521,6 @@ - (NSDictionary *)getChildArgs:(NSString *)name
     LP_END_TRY
 }
 
-/**
- * Prefix given event with all parent actionContext names to while filtering out the string
- * "action" (used in ExperimentVariable names but filtered out from event names).
- */
-- (NSString *)eventWithParentEventNamesFromEvent:(NSString *)event
```

Comment:
Let's keep this. It is easier to understand why we have it. "Used in ExperimentVariable" etc..